src/visual.ts

```diff
@@ -103,7 +108,21 @@ export default class TimelineStoryteller implements IVisual {
      */
     public update(options: VisualUpdateOptions): void {
         const dv = this.dataView = options.dataViews && options.dataViews[0];
+        
         if (dv && dv.categorical) {
+
+            const visualSelection = d3.select(".timeline_storyteller");
```

Comment:
replace .timeline_storyteller  by this.element

Reply:
Done